### ISSUE TITLE
Avocode bug, cant inspect elements

### ISSUE THREAD

**Issue by Exteriordemonic** [avocode/avocode#2217]
---
* **Version**: 3.6.0-web.2
* **Web/Desktop**: Web
* **Platform**: windows 10.0
* **Browser**: chrome 69.0.3497.100
* **Avocode UID**: 430405
---
```
TypeError: Cannot read property 'set' of undefined
    at https://cdn.app.avocode.com/dist/v-1560f71/js/avocode-web.min.js:156:1067315
    at e (https://cdn.app.avocode.com/dist/v-1560f71/js/avocode-web.min.js:1:15534)
    at e (https://cdn.app.avocode.com/dist/v-1560f71/js/avocode-web.min.js:1:15623)
    at e (https://cdn.app.avocode.com/dist/v-1560f71/js/avocode-web.min.js:1:15623)
    at Ue.updateIn (https://cdn.app.avocode.com/dist/v-1560f71/js/avocode-web.min.js:1:15687)
    at https://cdn.app.avocode.com/dist/v-1560f71/js/avocode-web.min.js:156:1067279
    at https://cdn.app.avocode.com/dist/v-1560f71/js/avocode-web.min.js:1:43905
    at https://cdn.app.avocode.com/dist/v-1560f71/js/avocode-web.min.js:1:37612
    at ft.__iterate (https://cdn.app.avocode.com/dist/v-1560f71/js/avocode-web.min.js:1:26039)
    at Pt.__iterate (https://cdn.app.avocode.com/dist/v-1560f71/js/avocode-web.min.js:1:37580)
    at vn.un.__iterate (https://cdn.app.avocode.com/dist/v-1560f71/js/avocode-web.min
```

**Comment by crashmajstr** [comment]
Hi @Exteriordemonic  , we're currently working on a fix, I am sorry for the inconvenience, we'll let you know once it's fixed.
Closing with respect to #2202
Best,
Josef

**Comment by crashmajstr** [comment]
Hi, just wanna inform you that we’ll release the fix in an upcoming hour or so. It’ll be version 3.6.1.
For now, you can just download and use our older stable build 3.5.6 

Download links
Mac: https://we.tl/t-bS4jyZufpT
Win: https://we.tl/t-12nDKmpafd
Linux: https://we.tl/t-krrbw7ZRvB

I’ll let you know once the fix is out. Sorry for the inconvenience.

**Comment by crashmajstr** [comment]
Hi, just wanna inform you that we’ve just released the new version of Avocode 3.6.1 containing the fix for this issue. 
Please, update your desktop app, if you use the web app the new version is already there.
Hope everything will be ok from now on.
I am very sorry for the inconvenience. 
On behalf of Avocode,
Josef Kettner
